.github/workflows/bumpdevversion.yml

```
@@ -0,0 +1,115 @@
+on:
+  pull_request:
+    branches:
+      - devel
+  # Schedule runs on 9am every Sunday
+  schedule:
+    - cron: '0 9 * * 0'
```

```suggestion
  # Schedule runs on 9am every Tuesday
  schedule:
    - cron: '0 9 * * 2'
```

Since this won't be an automated merge, shall we go with [Tuesday](https://en.wikipedia.org/wiki/Patch_Tuesday) instead of Sunday?

---

Ok, changed to Tuesday.